templates/topic.tera.html

```diff
@@ -8,25 +8,32 @@ <h1>{{ topic.name }}</h1>
 <p>{{ topic.description }}</p>
 
 {% for project in projects %}
-    <h2>{{ project.name }}</h2>
+    <h2>
+        {{ project.name }}
+        {% if project.docs %}
+            <a title="documentation" href="{{ project.docs }}"><i class="fa fa-home"></i></a>
+        {% endif %}
+        {% if project.repo %}
+            <a title="repository" href="{{ project.repo }}"><i class="fa fa-code"></i></a>
+        {% endif %}
+    </h2>
 
     {% if project.description %}
         <p>{{ project.description }}</p>
     {% endif %}
     
+    {% if project.crates     > 1 %}
```

Comment:
```suggestion
    {% if project.crates %}
```
This PR won't build on my PC without this change.

Reply:
https://github.com/arewelangyet/arewelangyet/pull/18/commits/d4cc854d89c2ff81be6a8fe1a9a83270d44d676e